examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>In the last line before "Solving this differential equation", I'm pretty sure it should be:</p><p> </p><pre class="ql-syntax" spellcheck="false">\frac{1}{S(t)} \cdot \lim_{\Delta t \to 0} \frac{S(t) - S(t + \Delta t)}{\Delta t}
</pre><p>instead of</p><pre class="ql-syntax" spellcheck="false">\frac{1}{S(t)} \cdot \lim_{\Delta t \to 0} \frac{S(t + \Delta t) - S(t)}{\Delta t}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>